devenv/dev-dashboards/panel-timeline/timeline-demo.json

```diff
@@ -168,9 +177,17 @@
       "options": {
         "alignValue": "center",
         "colWidth": 1,
+        "legend": {
+          "displayMode": "list",
+          "placement": "bottom"
+        },
+        "mergeValues": true,
         "mode": "changes",
```

Comment:
Note that the `mode` here _is_ appearing sometimes, despite it not being intended to be part of the persisted data model. Something to fix in the frontend.